toolbox/org.lamport.tla.toolbox/src/org/lamport/tla/toolbox/util/RCPNameToFileIStream.java

```diff
@@ -225,6 +225,10 @@ public String getFullPath() {
 		}
 		return buf.toString();
 	}
+
+	public String getLibraryPath(String name) {
+		return "";
+	  }
```

Comment:
The RCPNameToFileIStream is used by the Toolbox.  The T. definetly supports libary path locations. Why do this method return the emtpy string?

Reply:
1.) TLC CLI => SimpleFilenameToStream
2.) Toolbox => RCPNameToFileIStream
3. ) Distributed tLC => RMIFilenameToStreamResolver